src/api/types.ts

```diff
@@ -13094,13 +13094,13 @@ export interface RollupRollupSearchRequest extends RequestBase {
   size?: integer
 }
 
-export interface RollupRollupSearchResponse<TDocument = unknown> {
+export interface RollupRollupSearchResponse<TDocument = unknown, TAggregations = Record<AggregateName, AggregationsAggregate>> {
```

Comment:
Shouldn't we enforce `TAggregations`  abide by  `Record<AggregateName, AggregationsAggregate>` interface? 
`TAggregations extends Record<AggregateName, AggregationsAggregate> = Record<AggregateName, AggregationsAggregate>`?
However,`AggregationsAggregate` is a union of types, I'm not sure it won't disrupt API consumers' code. We can do it in a follow-up.